docs/5.0/application-access.md

```diff
@@ -0,0 +1,301 @@
+
+# Teleport Application Access
+
+## Introduction
+Teleport Application Access has been designed to provide secure access to internal dashboards and applications. Building on Teleports strong foundations of security and identity. You can now put applications onto the internet safely and securely.
+
+You can secure any web application using application access:
+
+* Internal control panels
+* Wikis / Tooling that's only available on the VPN.
+* Infra dashboards - Kubernetes, Grafana
+* Developer tools, such as Jenkins, Gitlab or Ops genie
+
+
+#### Hardware Requirements
+[How it works](https://gravitational.com/teleport/how-it-works/) page will give you a good overview of the setup.
+
+#### Networking Requirements
+
+!!! note "Why do I see SSH Ports for Application Access?"
+
+    Teleport uses SSH reverse tunnels to connect applications to the proxy. This is why configuration below mentions SSH ports.
+
+|Port      | Service    | Description
+|----------|------------|-------------------------------------------
+|3023      | Proxy      | SSH port for clients who need tsh login
+|3024      | Proxy      | SSH port used to create reverse SSH tunnels from behind-firewall environments into a trusted proxy server.
+|3025      | Auth       | TLS port used by the Auth Service to serve its API to other nodes in a cluster.
+|443       | Proxy      | HTTPS connection to authenticate `tsh` users and web users into the cluster. The same connection is used to serve a Teleport UI.
+
+#### TLS Requirements
+
+TLS is required to secure Teleports Unified Access Plane and any connected applications. When setting up Teleport the minium requirement is a certificate for the proxy and a wild card certificate for it's sub-domain. This is where everyone will login to Teleport.
+
+Example: `teleport.example.com` will host the Access Plane. `*.teleport.example.com` will host all of the applications. e.g. `jenkins.teleport.example.com`. Teleport supports accessing these applications on other domains if required. Both DNS and the correct certificates need to be obtained for this to work.
+
+```yaml
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  tunnel_listen_addr: 0.0.0.0:3024
+  public_addr: teleport.example.com:443
+  ## Example of two https keypairs.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/teleport.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+```
+
+!!! tip "Using Certbot to obtain Wildcard Certs"
+
+    Let's Encrypt provides free wildcard certificates. If using [certbot](https://certbot.eff.org/)
+    with DNS challenge the below script will make setup easy.
+
+      ```sh
+      certbot certonly --manual \
+        --preferred-challenges=dns \
+        --email [EMAIL] \
+        --server https://acme-v02.api.letsencrypt.org/directory \
```

Comment:
I don't think we need to specify the server at all, `certbot` will handle that automatically.

```suggestion
```